content/c++/encyclopedia/exceptions-in-cpp.md

```diff
@@ -0,0 +1,133 @@
+---
+Title: "Exceptions in C++"
+Subjects:
+  - "Computer Science"
+  - "Game Development"
+Tags:
+  - "Catch"
+  - "Errors"
+  - "Try"
+Catalog Content:
+  - "https://www.codecademy.com/learn/learn-c-plus-plus"
+  - "https://www.codecademy.com/learn/paths/computer-science"
+  - "https://github.com/Codecademy-Curriculum/codepedia-content/blob/main/content/language-agnostic/encyclopedia/what-is-an-error.md"
```

Comment:
Let's take out the URL for `what-is-an-error.md`. We shouldn't link to other entries in the metadata, since the URLs could change.